eng/Versions.props

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27324-5</MicrosoftNETCoreAppPackageVersion>
     <MicrosoftWin32RegistryPackageVersion>4.6.0-preview.19073.11</MicrosoftWin32RegistryPackageVersion>
+    <MicrosoftBclJsonSourcesPacakgeVersion>4.6.0-preview.19073.11</MicrosoftBclJsonSourcesPacakgeVersion>
```

Comment:
To add this to dependency flow, you also need to add an entry to eng/Versions.Details.xml. Without an entry in Versions.Details.xml, automation won't update this value.

Reply:
By the way, I've written step by step instructions what what to do now when adding new dependencies. See https://github.com/aspnet/Extensions/blob/master/docs/ReferenceResolution.md#example-adding-a-new-dependency